inprocess/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -111,6 +111,7 @@ public static String generateName() {
   private final ServerImplBuilder serverImplBuilder;
   final SocketAddress listenAddress;
   int maxInboundMetadataSize = Integer.MAX_VALUE;
+  long assumedMessageSize = -1;
```

Comment:
Does this do anything?

Reply:
This value is looked for to identify that the assumedMessageSize hasn't been set.  It would be nice to, as was said above, consider any negative number to be disabling assumedMessageSize and allow users to specify negative numbers (with appropriate update to javadoc).

Reply:
Let me say this another way: it seems either the client-side or server-side value is ignored. InProcessTransport is shared between client and server, so one must be thrown away. I thought it was server-side, but looking a bit more I now see the `assumedMessageSize = server.getAssumedMessageSize();` line in start(), maybe it is client-side as the value is overwritten by server-side.